src/restapi/fTarget.ts

```diff
@@ -17,15 +17,14 @@ export const fCreateTarget: CallResponseHandler = async (req, res) => {
         mattermostSiteUrl: context.mattermost_site_url,
     };
     const zdClient = await newZDClient(zdOptions);
-    let callResponse: AppCallResponse;
-    try {
-        const text = await updateOrCreateTarget(zdClient, context);
-        callResponse = newOKCallResponseWithMarkdown(text);
-        res.json(callResponse);
-    } catch (error) {
-        callResponse = newErrorCallResponseWithMessage('Unable to create target: ' + error.message);
-        res.json(callResponse);
-    }
+    await tryCallResponseWithMessage(
+        updateOrCreateTarget(zdClient, context).then((text) => {
+            const callResponse = newOKCallResponseWithMarkdown(text);
+            res.json(callResponse);
+        }),
+        'Unable to create target',
+        res
+    );
```

Comment:
What do you think about avoiding the usage of `then()` when calling 
`tryCallResponseWithMessage`? I think this looks cleaner:

```suggestion
    const text = await tryCallResponseWithMessage(
        updateOrCreateTarget(zdClient, context),
        'Unable to create target',
        res
    );

    const callResponse = newOKCallResponseWithMarkdown(text);
    res.json(callResponse);
```

I do like the multiple-line approach here though. It's clear that there is special error handling going on this way.

Reply:
That's much cleaner. I'll go through and make similar changes. Thank you for teaching me about codebase conventions as well.